std/string.d

```diff
@@ -484,6 +484,20 @@ ptrdiff_t indexOf(Range)(Range s, in dchar c,
     return -1;
 }
 
+///
+unittest
+{
+    string s = "Hello World";
+    auto i = indexOf(s, 'W');
+    assert(i == 6);
```

Comment:
string s is used to mitigate copy paste erros.

The other part is personal style, I guess.